Update vsphere operations guide page (#700)
portworx/pxdocs: content/cloud-references/auto-disk-provisioning/vsphere/operations-guide.md

## Capacity Management

Increasing capacity falls into 2 primary buckets:

### a. Increase storage capacity for a storage pool

> As storage usage increases, you will need to expand the Portworx storage pools. 

Author: This doc aims to be a vsphere focused operations guide. Expanding storage pools is just part of it.

We can link the more detailed page you referenced above in this PR. Can you find a good place?

## Capacity Management

You can increase capacity in one of two ways:

### A. Increase storage capacity for a storage pool

As storage usage increases, you must expand the Portworx storage pools:

1. Identify the Portworx storage pool you want to expand by [listing them](#listing-storage-pools-created-by-portworx). 
2. Enter the `pxctl service pool expand` command specifying the following:

     * UUID: The UUID of the pool you want to expand. This is found the **POOL** column when you listed the pools.
     * SIZE: The minimum new required size of the storage pool in GiB.

>      * OPERATION-TYPE: The type of disk operation that will be performed to expand the pool. You have 2 options 

Reviewer: ```suggestion
     * OPERATION-TYPE: The type of disk operation that will be performed to expand the pool. You have 2 options:
```

         * **resize-disk**: Portworx will resize existing VMDKs in the storage pool. This is the recommended operation as this does not require data movement on the backing drives in the pool.

>          * **add-disk**: Portworx will create one or more VMDKs based on the required new size of the pool. After the drive is added to the pool, Portworx will rebalance the volumes onto the new VMDKs. Hence this operation can take a while.

Reviewer: ```suggestion
         * **add-disk**: Portworx will create one or more VMDKs based on the required new size of the pool. After the drive is added to the pool, Portworx will rebalance the volumes onto the new VMDKs. As a result, this operation can take a while to complete.
```

>     ```text

Reviewer: ```suggestion

    ```text
```
